validation/views.py

```diff
@@ -163,6 +164,19 @@ def add_cors_headers(response):
     return response
 
 
+def segment_content_view(request, segment_id):
+    """
+    The view for a single segment
+    Returns the selected phrase and info provided by the helper functions
+    """
+    phrases = Phrase.objects.filter(id=segment_id)
+    segment_name = phrases[0].transcription
+    suggestions = get_distance_with_translations(segment_name)
+    context = dict(phrases=phrases, segment_name=segment_name, suggestions=suggestions)
```

Comment:
I think there is a standard way for pages to get auth information but I don’t remember it off-hand. That would be great; this will work for now, and won’t show the “Login” link when already logged in.

```suggestion
    auth = request.user.is_authenticated
    context = dict(phrases=phrases, segment_name=segment_name, suggestions=suggestions, auth=auth)
```

Reply:
I forgot that I added the auth part to the other views, nice catch